rusk/Makefile

```diff
@@ -2,10 +2,10 @@ help: ## Display this help screen
 	@grep -h -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-15s\033[0m %s\n", $$1, $$2}'
 
 keys: ## Build circuit keys
-	cargo build --release
+	RUSK_PROFILE_PATH=./../ cargo build --release
```

Comment:
I don't understand what is the meaning of this but I would leave this option to the user. If he wants to use custom directories for the rusk keys, then he can:

`$ RUSK_PROFILE_PATH=/tmp/rusk make keys`

Suggestion: rollback this line